modules/ROOT/pages/apikit-4-xml-reference.adoc

```diff
@@ -1,91 +1,97 @@
-= APIkit XML Reference
+= APIkit for REST Module Reference
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: apikit, maven
 
-This reference contains dependency, namespace, schema, and other information about APIkit.
+Learn the fundamentals about dependency, namespace, schema, and other information about APIkit for REST.
 
-== APIkit Dependency Information
+== Dependency Information
 
 ----
 <dependency>
   <groupId>org.mule.modules</groupId>
   <artifactId>mule-apikit-module</artifactId>
-  <version>1.1.0</version>
+  <version>LATEST_VERSION</version> //<!--1-->
   <classifier>mule-plugin</classifier>
 </dependency>
 ----
+[calloutlist]
+.. Check https://docs.mulesoft.com/release-notes/apikit/apikit-release-notes[APIkit release notes] to get the *LATEST_VERSION*.
 
-*APIkit Namespace and Schema*
+== Namespace and Schema
 
 ----
-xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
-xsi:schemaLocation="http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd"
+xmlns:apikit="http://www.mulesoft.org/schema/mule/mule-apikit"
+xsi:schemaLocation="http://www.mulesoft.org/schema/mule/mule-apikit http://www.mulesoft.org/schema/mule/mule-apikit/current/mule-apikit.xsd"
 ----
 
-=== apikit Elements
+== Configuration and Operations
 
 [%header,cols="30a,50a,8a"]
 |===
 |Element Name |Description |Required
-|*apikit:config* |APIkit configuration |Y
-|*apikit:console* |Refers to the config-ref property in the http:listener element in the api-console flow that defines the API Console URL |N
-|*apikit:router* |A rest routing message processor nested in the flow element |Y
+|`apikit:config` |APIkit configuration |Y
+|`apikit:console` |`config-ref` property in the `http:listener` element in the api-console flow that defines the API console URL |N
+|`apikit:router` |REST routing message processor nested in the flow element |Y
 |===
 
-=== apikit:config Attributes
+=== `apikit:config` Attributes
 
-These elements are of type NonBlankString.
+The `apikit:config` attributes are of `NonBlankString`type`.
 
 [%header,cols="22a,10a,68a"]
 |===
 |Attribute Name|Required |Description
-|*outboundHeadersMapName* |N |Used to identify the name of the map the contains response headers
-|*httpStatusVarName* |N |The name of the variable that stores the HTTP status code in the response
-|*keepRamlBaseUri* |N | An alias for the equivalent "keepApiBaseUri" attribute.
+|`outboundHeadersMapName` |N |Name of the map the contains response headers.
```

Comment:
```suggestion
|`outboundHeadersMapName` |N |Name of the map the contains response headers
```

Reply:
Remove terminating punctuation if none are full sentences.

Reply:
Disregarding this comment as suggested by you ;) 
Thanks!